src/executables/master_server_monitor.hpp

```diff
@@ -26,31 +26,34 @@
 #include <cstring>
 #include <thread>
 #include <vector>
+#include <list>
 #include <memory>
 #include <time.h>
 #include "connection_defs.h"
+#include "connection_globals.h"
 
 class server_monitor
 {
   public:
     class server_entry
     {
       public:
-	server_entry (int pid, const char *server_name, const char *exec_path, char *args, CSS_CONN_ENTRY *conn);
+	server_entry (int pid, const char *exec_path, char *args, CSS_CONN_ENTRY *conn);
 	~server_entry () {};
 
-	server_entry (const server_entry &) = delete;
+	server_entry (const server_entry &) = default;
 	server_entry (server_entry &&) = delete;
 
-	server_entry &operator = (const server_entry &) = delete;
-	server_entry &operator = (server_entry &&) = delete;
+	server_entry &operator= (const server_entry &) = delete;
+	server_entry &operator= (server_entry &&) = default;
+
+	CSS_CONN_ENTRY *get_conn ();
```

Comment:
```suggestion
	CSS_CONN_ENTRY *get_conn () const;
```
객체 상태를 변경하지 않는 getter 의 경우 const 함수로 정의하는게 좋아보입니다.

Reply:
수정하였습니다.